staff/aleix-caminal/app-template/about.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>About</title>
+    </head>
+    <body>
+        <header>
+            <h1>About</h1><hr>
+        </header>
+        <main>
+            <section id="us">
+                <h2>Aleix Caminal</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </section>
+            <section id="services">
```

Comment:
why ids? are they really necessary? think about it

Reply:
They allow you to navigate between sections and differentiate them, I find them useful